CRM/Price/BAO/LineItem.php

```diff
@@ -1101,6 +1103,28 @@ protected static function getTaxAmountForLineItem(array $params): float {
     return ($taxRate / 100) * $params['line_total'];
   }
 
+  /**
+   * Do we need to allow for the possibility an extension is try to force it's own tax through.
+   *
+   * This extra if is a temporary fix for https://lab.civicrm.org/dev/core/-/issues/2796
+   * discussion to continue on a longer fix.
+   *
+   * If an extension is setting tax rate for a financial type with no configured tax rate then
+   * let it until we figure out a more durable solution.
+   *
+   * @param array $params
+   */
+  protected static function needsToPermitExtensionHackery($params): bool {
+    $taxRates = CRM_Core_PseudoConstant::getTaxRates();
+    if (isset($taxRates[$params['financial_type_id']])) {
+      return FALSE;
```

Comment:
I am not sure this is going to work because you need Sales tax account to be configured for financial type to make Tax work, that means if you have Sales tax account linked to financial type, CRM_Core_PseudoConstant::getTaxRates() will return the financial type.

Reply:
@pradpnayak OK - I was just looking at what the code looked like before the change you pointed out as having caused issues